src/playground/index.html

```diff
@@ -10,13 +10,26 @@
     <!-- Stub transcendInit -->
     <script>
       if (!self.transcend) {
-        self.transcend = {
+        /**
```

Comment:
What is this for? afaik this whole PR could be a single-line functional change (ignoring types)